index.html

```diff
@@ -416,60 +402,6 @@ <h4 id="additional-definitions-">Additional Definitions:</h4>
       <dd>
         The Web of Things Thing Description as defined in [[wot-thing-description]].
       </dd>
```

Comment:
Agree to remove.

Reply:
https://github.com/w3c/wot-profile/pull/116